src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -88,6 +88,8 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
 
         } else if (batch.Batch_Gift_Entry_Version__c >= 2.0) {
 
+            fieldApiNames.add(DataImport__c.Additional_Object_JSON__c.getDescribe().getName());
```

Comment:
Bringing over comment from Ben: "We don't have the ability to change this yet, but we should retrieve this field name from the Custom Settings Facade to future-proof this."